### PR TITLE
Source Asana: Add dbt converter

### LINKED ISSUE #9: Create dbt converter for Airbyte Asana source connector
The goal of this task is to ensure the Airbyte Asana connector output can be used to run [Fivetran's dbt analytical model](https://hub.getdbt.com/fivetran/asana/latest/).

**Definition of Done:** 
* Create a pull request that adds new tables or views, enabling the execution of Fiventran's dbt with Airbyte output data.
* Update the documentation inside the connector folder to explain how to configure the user dbt package to run the analytical model.

**Resources:** 
* Check example of how it was done for [Zendesk Support](link) connector

### PR DESCRIPTION
**Pull request by pangeran-bottor**
closes #9

### CODE REVIEW

File: connectors/source_asana/dbt_project.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ config-version: 2
 
 version: 0.1.0
 
-profile: airbyte
+profile: integration_tests
```
Comment (marcosmarxm) [inline connectors/source_asana/dbt_project.yml:7]: keep profile as airbyte

File: connectors/source_asana/integration_tests/dbt_project.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+vars:
+  # Required by Airbyte dbt model
+  using_fivetran_model: True
+  airbyte_database: "airbyte_db_default"
+  airbyte_schema: "airbyte_dbt_asana"
+
+  # Required by Fivetran dbt model
+  asana_source:
+    asana_database: "airbyte_db_default"
+    asana_schema: "airbyte_dbt_asana"
```
Comment (marcosmarxm) [inline connectors/source_asana/integration_tests/dbt_project.yml:44]: keep same level as the other variables.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: connectors/source_asana/models/fivetran_converter/project.sql [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            t.modified_at,
+            t.name,
+            JSON_QUERY(t.owner, '$.gid') as owner_id,
+            t.public,
+            JSON_QUERY(t.team, '$.gid') as team_id,
+            JSON_QUERY(t.workspace, '$.gid') as workspace_id,
+            t.notes
+        FROM
+            {{ source('source_asana', 'projects') }} t
+    )
```
Comment (marcosmarxm) [inline connectors/source_asana/models/fivetran_converter/project.sql:20]: Nice start @pangeran-bottor you can use Chatgpt to translate to snowflake and postgres.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Comment by marcosmarxm** [comment]
Nice @pangeran-bottor I'm gonna review tomorrow! Sorry the delay

**Comment by marcosmarxm** [comment]
@pangeran-bottor it looks good, missing a couple of models.

**Comment by pangeran-bottor** [comment]
Thanks for the help @marcosmarxm 
Finalized this PR to be reviewed. The models already cover all Fivetran models from https://github.com/fivetran/dbt_asana_source/blob/main/models/src_asana.yml

**Review by marcosmarxm — APPROVED** [review]
Thanks a lot @pangeran-bottor the model is working now :)

**Pull request merged**